### ISSUE TITLE
Unable to see every dispatch rule at  service on the customer contract  tab

### ISSUE THREAD

**Issue by pmendoza12** [Super-Visions/sv-auto-dispatch#2]
After installing The module `sv-auto-dispatch`,  i can see the Dispatch rules menu, and I can create a new one! . But I don´t  know where to assign this rules to the Services on the customer contact, i can not see any tab or option where to associate the dispatch rule to the Services.

this is my configuration and the version instalation...
```
iTopVersion: 2.7.0-1
iTopBuild: 5541
iTopBuildDate: 2020-04-01 17:42:11
DataModelVersion: 2.7.0
MySQLVersion: 8.0.17
PHPVersion: 7.2.15
OSVersion: WINNT
WebServerVersion: Microsoft-IIS/8.5
PHPModules: Core, bcmath, calendar, ctype, date, filter, hash, iconv, json, SPL, pcre, readline, Reflection, session, standard, mysqlnd, tokenizer, zip, zlib, libxml, dom, PDO, bz2, SimpleXML, xml, wddx, xmlreader, xmlwriter, cgi-fcgi, imap, curl, gd, gettext, mysqli, mbstring, openssl, soap, xmlrpc, ldap, Phar, com_dotnet, sqlite3, odbc
ItopSetting/cron_max_execution_time: 600
ItopSetting/timezone: America/La_Paz
PHPSetting/memory_limit: 128M
PHPSetting/max_execution_time: 300
PHPSetting/upload_max_filesize: 32M
PHPSetting/post_max_size: 32M
MySQLSetting/max_allowed_packet: 104857600
MySQLSetting/key_buffer_size: 33554432
MySQLSetting/query_cache_size:
MySQLStatus/Key_read_requests: 2317
MySQLStatus/Key_reads: 37
InstallDate: 2020-07-13 14:07:07
InstallPath: E:\inetpub\wwwroot\iTop\web/
---- Installation choices ----
InstalledExtension/itop-config-mgmt-core/2.7.0-1
InstalledExtension/itop-config-mgmt-datacenter/2.7.0-1
InstalledExtension/itop-config-mgmt-end-user/2.7.0-1
InstalledExtension/itop-config-mgmt-storage/2.7.0-1
InstalledExtension/itop-config-mgmt-virtualization/2.7.0-1
InstalledExtension/itop-service-mgmt-enterprise/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-user-request/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-incident/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-enhanced-portal/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil/2.7.0-1
InstalledExtension/itop-change-mgmt-itil/2.7.0-1
InstalledExtension/itop-kown-error-mgmt/2.7.0-1
InstalledExtension/itop-problem-mgmt/2.7.0-1
InstalledExtension/combodo-autoclose-ticket/1.0.1 (Extension)
InstalledExtension/contact_ticket/1.0.0 (Extension)
InstalledExtension/fieldproject1-module/1.0.0 (Extension)
InstalledExtension/fieldservicefamily-module/1.0.0 (Extension)
InstalledExtension/ldap-data-collector/1.2.0 (Extension)
InstalledExtension/samplefield_inciden-module/1.0.0 (Extension)
InstalledExtension/sv-auto-dispatch/1.0.1 (Extension)
InstalledExtension/WorkOrderTemplate/1.0.0 (Extension)
InstalledExtension/combodo-approval-process-automation/2.0.1 (iTop Hub)
InstalledExtension/combodo-configurator-for-automatic-object-creation/1.1.2 (iTop Hub)
InstalledExtension/combodo-customized-request-forms/2.0.14 (iTop Hub)
InstalledExtension/combodo-db-tools/1.0.7 (iTop Hub)
InstalledExtension/combodo-dispatch-incident/1.1.4 (iTop Hub)
InstalledExtension/combodo-dispatch-userrequest/1.1.5 (iTop Hub)
InstalledExtension/combodo-mail-to-ticket-automation/3.1.0 (iTop Hub)
InstalledExtension/combodo-predefined-response-models/1.1.4 (iTop Hub)
InstalledExtension/customer-survey/2.4.2 (iTop Hub)
InstalledExtension/combodo-send-updates-by-email/1.2.0 (iTop Hub)
InstalledExtension/itop-admin-delegation-profiles/0.1.0 (iTop Hub)
InstalledExtension/itop-communications/1.1.0 (iTop Hub)
InstalledExtension/combodo-user-actions-configurator/1.3.2 (iTop Hub)
InstalledExtension/combodo-data-synchronization-dashboard/1.0.0 (iTop Hub)
InstalledExtension/molkobain-console-tooltips/1.1.2 (iTop Hub)
InstalledExtension/molkobain-portal-mosaic-service-catalog/1.0.0 (iTop Hub)
InstalledExtension/combodo-sla-considering-business-hours/2.1.7 (Extension)
---- Actual modules installed ----
InstalledModule/approval-base: 3.0.1
InstalledModule/authent-cas: 2.7.0
InstalledModule/authent-external: 2.7.0
InstalledModule/authent-ldap: 2.7.0
InstalledModule/authent-local: 2.7.0
InstalledModule/combodo-autoclose-ticket: 1.0.1
InstalledModule/combodo-db-tools: 2.7.0
InstalledModule/combodo-email-synchro: 3.2.0
InstalledModule/itop-attachments: 2.7.0
InstalledModule/itop-backup: 2.7.0
InstalledModule/itop-config-mgmt: 2.7.0
InstalledModule/itop-config: 2.7.0
InstalledModule/itop-datacenter-mgmt: 2.7.0
InstalledModule/itop-endusers-devices: 2.7.0
InstalledModule/itop-files-information: 2.7.0
InstalledModule/itop-hub-connector: 2.7.0
InstalledModule/itop-ldap-collector: 1.0.1
InstalledModule/itop-object-copier: 1.3.2
InstalledModule/itop-portal-base: 2.7.0
InstalledModule/itop-portal: 2.7.0
InstalledModule/itop-profiles-itil: 2.7.0
InstalledModule/itop-sla-computation: 2.7.0
InstalledModule/itop-standard-email-synchro: 3.1.0
InstalledModule/itop-stencils: 1.0.6
InstalledModule/itop-storage-mgmt: 2.7.0
InstalledModule/itop-tickets: 2.7.0
InstalledModule/itop-virtualization-mgmt: 2.7.0
InstalledModule/itop-welcome-itil: 2.7.0
InstalledModule/molkobain-handy-framework: 1.3.0
InstalledModule/molkobain-newsroom-provider: 1.1.0
InstalledModule/molkobain-portal-mosaic-service-catalog: 1.0.0
InstalledModule/templates-base: 3.0.17
InstalledModule/contact_ticket: 1.0.0
InstalledModule/customer-survey: 2.4.2
InstalledModule/email-reply: 1.2.0
InstalledModule/itop-admin-delegation-profiles: 0.1.0
InstalledModule/itop-approval-portal: 1.0.4
InstalledModule/itop-bridge-virtualization-storage: 2.7.0
InstalledModule/itop-change-mgmt-itil: 2.7.0
InstalledModule/itop-core-update: 2.7.0
InstalledModule/itop-incident-mgmt-itil: 2.7.0
InstalledModule/itop-knownerror-mgmt: 2.7.0
InstalledModule/itop-problem-mgmt: 2.7.0
InstalledModule/itop-request-mgmt-itil: 2.7.0
InstalledModule/itop-service-mgmt: 2.7.0
InstalledModule/itop-synchro-dashboard: 1.0.0
InstalledModule/molkobain-console-tooltips: 1.1.2
InstalledModule/precanned-replies: 1.1.5
InstalledModule/samplefield_inciden-module: 1.0.0
InstalledModule/WorkOrderTemplate: 1.0.0
InstalledModule/combodo-dispatch-incident: 1.1.4
InstalledModule/combodo-dispatch-userrequest: 1.1.5
InstalledModule/combodo-sla-computation: 2.3.0
InstalledModule/fieldproject1-module: 1.0.0
InstalledModule/fieldservicefamily-module: 1.0.0
InstalledModule/itop-communications: 1.1.0
InstalledModule/itop-full-itil: 2.7.0
InstalledModule/itop-request-template: 2.0.5
InstalledModule/precanned-replies-pro: 1.0.5
InstalledModule/sv-auto-dispatch: 1.0.1
InstalledModule/combodo-approval-extended: 1.3.0
InstalledModule/combodo-coverage-windows-computation: 2.0.2
InstalledModule/itop-request-template-portal: 1.0.0
```

**Comment by Hipska** [comment]
Could it be possible to replace your info with what is to be found under "About iTop" and then open the section "Support information". And please also put it in code tags or as a quote.

But they can be found as setting on the link between CustomerContract and Service:

- On a CustomerContract go to the tab of Services.
- On a Service, go to the tab of CustomerContracts.

It could be possible that one of the other modules you installed might hide the field, you can unhide it for yourself by configuring the list. If you can find out which module causes this, then you could report that to the owner/creator of it.

**Comment by pmendoza12** [comment]
Hi... thanks for answer me back...
Here the information toy request

About Itop:
iTop, © 2010-2018 Combodo SARL is licensed under the AGPL v3 license. (Details)
jQuery, jQuery UI, © the jQuery Foundation is licensed under the MIT license. (Details)
The jQuery tooltip plugin, © Craig Thompson is licensed under the MIT license. (Details)
jQuery Hotkeys Plugin, © John Resig is licensed under the MIT or GPL Version 2 licenses license. (Details)
Swift Mailer, © Fabien Potencier is licensed under the MIT license. (Details)
jQuery-File-Upload, © Sebastian Tschan is licensed under the MIT license. (Details)
PHP XLSXWriter, © Mark Jones is licensed under the MIT license. (Details)
TCPDF, © Nicola Asuni is licensed under the LGPL license. (Details)
fraphael, © Chris Scott is licensed under the MIT license. (Details)
Silex, © Fabien Potencier is licensed under the MIT license. (Details)
Bootstrap, © Twitter is licensed under the MIT license. (Details)
Font Awesome, © Dave Gandy is licensed under the MIT license. (Details)
DataTables, © SpryMedia Ltd is licensed under the MIT license. (Details)
jQuery Timepicker addon, © Trent Richardson is licensed under the MIT license. (Details)
D3 js, © Mike Bostock is licensed under the BSD license. (Details)
C3 js, © Masayuki Tanaka is licensed under the MIT license. (Details)
CKEditor, © CKSource - Frederico Knabben is licensed under the LGPL license. (Details)
scssphp, © Leaf Corcoran is licensed under the MIT license. (Details)
Emogrifier, © Pelago is licensed under the MIT license. (Details)
Archive_Tar, © Vincent Blavet is licensed under the BSD license. (Details)
Selectize.js, © Brian Reavis is licensed under the Apache License v2 license. (Details)
composer, © Nils Adermann, Jordi Boggiano is licensed under the MIT license. (Details)
Droid Sans Fallback font, © The Android Open Source Project is licensed under the Apache License 2.0 license. (Details)
font-awesome, © Travis Chase Dave Gandy Rob Madole Jory Raphael Geremia Taglialatela Brian Talbot Mike Wilkerson is licensed under the GPL license. (Details)
apereo/phpcas, © Joachim Fritschi - Adam Franco is licensed under the Apache-2.0 license. (Details)
ajaxorg/ace, © Ajax.org B.V. is licensed under the BSD 3-clause "New" or "Revised" license. (Details)
clipboard.js, © Zeno Rocha is licensed under the MIT license. (Details)
combodo/tcpdf, © Nicola Asuni - Combodo is licensed under the LGPL-3.0-only license. (Details)
nikic/php-parser, © Nikita Popov is licensed under the BSD-3-Clause license. (Details)
paragonie/random_compat, © Paragon Initiative Enterprises is licensed under the MIT license. (Details)
pear/archive_tar, © Vincent Blavet - Greg Beaver - Michiel Rook is licensed under the BSD-3-Clause license. (Details)
pear/console_getopt, © Andrei Zmievski - Stig Bakken - Greg Beaver is licensed under the BSD-2-Clause license. (Details)
pear/pear-core-minimal, © Christian Weiske is licensed under the BSD-3-Clause license. (Details)
pear/pear_exception, © Helgi Thormar - Greg Beaver is licensed under the BSD-2-Clause license. (Details)
pelago/emogrifier, © Oliver Klee - Zoli Szabó - John Reeve - Jake Hotson - Cameron Brooks - Jaime Prado is licensed under the MIT license. (Details)
psr/cache, © PHP-FIG is licensed under the MIT license. (Details)
psr/container, © PHP-FIG is licensed under the MIT license. (Details)
psr/log, © PHP-FIG is licensed under the MIT license. (Details)
psr/simple-cache, © PHP-FIG is licensed under the MIT license. (Details)
scssphp/scssphp, © Anthon Pang - Cédric Morin is licensed under the MIT license. (Details)
swiftmailer/swiftmailer, © Chris Corbyn - Fabien Potencier is licensed under the MIT license. (Details)
symfony/cache, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/class-loader, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/config, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/console, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/css-selector, © Fabien Potencier - Jean-François Simon - Symfony Community is licensed under the MIT license. (Details)
symfony/debug, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/dependency-injection, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/dotenv, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/event-dispatcher, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/filesystem, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/finder, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/framework-bundle, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/http-foundation, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/http-kernel, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-apcu, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-ctype, © Gert de Pagter - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-mbstring, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-php56, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-php70, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/polyfill-util, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/routing, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/stopwatch, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/twig-bridge, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/twig-bundle, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/var-dumper, © Nicolas Grekas - Symfony Community is licensed under the MIT license. (Details)
symfony/web-profiler-bundle, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
symfony/yaml, © Fabien Potencier - Symfony Community is licensed under the MIT license. (Details)
twig/twig, © Fabien Potencier - Twig Team - Armin Ronacher is licensed under the BSD-3-Clause license. (Details)


Support Information:
===== begin =====
iTopVersion: 2.7.0-1
iTopBuild: 5541
iTopBuildDate: 2020-04-01 17:42:11
DataModelVersion: 2.7.0
MySQLVersion: 8.0.17
PHPVersion: 7.2.15
OSVersion: WINNT
WebServerVersion: Microsoft-IIS/8.5
PHPModules: Core, bcmath, calendar, ctype, date, filter, hash, iconv, json, SPL, pcre, readline, Reflection, session, standard, mysqlnd, tokenizer, zip, zlib, libxml, dom, PDO, bz2, SimpleXML, xml, wddx, xmlreader, xmlwriter, cgi-fcgi, imap, curl, gd, gettext, mysqli, mbstring, openssl, soap, xmlrpc, ldap, Phar, com_dotnet, sqlite3, odbc
ItopSetting/cron_max_execution_time: 600
ItopSetting/timezone: America/La_Paz
PHPSetting/memory_limit: 128M
PHPSetting/max_execution_time: 300
PHPSetting/upload_max_filesize: 32M
PHPSetting/post_max_size: 32M
MySQLSetting/max_allowed_packet: 104857600
MySQLSetting/key_buffer_size: 33554432
MySQLSetting/query_cache_size: 
MySQLStatus/Key_read_requests: 2317
MySQLStatus/Key_reads: 37
InstallDate: 2020-07-13 14:07:07
InstallPath: E:\inetpub\wwwroot\iTop\web/
---- Installation choices ----
InstalledExtension/itop-config-mgmt-core/2.7.0-1
InstalledExtension/itop-config-mgmt-datacenter/2.7.0-1
InstalledExtension/itop-config-mgmt-end-user/2.7.0-1
InstalledExtension/itop-config-mgmt-storage/2.7.0-1
InstalledExtension/itop-config-mgmt-virtualization/2.7.0-1
InstalledExtension/itop-service-mgmt-enterprise/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-user-request/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-incident/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil-enhanced-portal/2.7.0-1
InstalledExtension/itop-ticket-mgmt-itil/2.7.0-1
InstalledExtension/itop-change-mgmt-itil/2.7.0-1
InstalledExtension/itop-kown-error-mgmt/2.7.0-1
InstalledExtension/itop-problem-mgmt/2.7.0-1
InstalledExtension/combodo-autoclose-ticket/1.0.1 (Extension)
InstalledExtension/contact_ticket/1.0.0 (Extension)
InstalledExtension/fieldproject1-module/1.0.0 (Extension)
InstalledExtension/fieldservicefamily-module/1.0.0 (Extension)
InstalledExtension/ldap-data-collector/1.2.0 (Extension)
InstalledExtension/samplefield_inciden-module/1.0.0 (Extension)
InstalledExtension/sv-auto-dispatch/1.0.1 (Extension)
InstalledExtension/WorkOrderTemplate/1.0.0 (Extension)
InstalledExtension/combodo-approval-process-automation/2.0.1 (iTop Hub)
InstalledExtension/combodo-configurator-for-automatic-object-creation/1.1.2 (iTop Hub)
InstalledExtension/combodo-customized-request-forms/2.0.14 (iTop Hub)
InstalledExtension/combodo-db-tools/1.0.7 (iTop Hub)
InstalledExtension/combodo-dispatch-incident/1.1.4 (iTop Hub)
InstalledExtension/combodo-dispatch-userrequest/1.1.5 (iTop Hub)
InstalledExtension/combodo-mail-to-ticket-automation/3.1.0 (iTop Hub)
InstalledExtension/combodo-predefined-response-models/1.1.4 (iTop Hub)
InstalledExtension/customer-survey/2.4.2 (iTop Hub)
InstalledExtension/combodo-send-updates-by-email/1.2.0 (iTop Hub)
InstalledExtension/itop-admin-delegation-profiles/0.1.0 (iTop Hub)
InstalledExtension/itop-communications/1.1.0 (iTop Hub)
InstalledExtension/combodo-user-actions-configurator/1.3.2 (iTop Hub)
InstalledExtension/combodo-data-synchronization-dashboard/1.0.0 (iTop Hub)
InstalledExtension/molkobain-console-tooltips/1.1.2 (iTop Hub)
InstalledExtension/molkobain-portal-mosaic-service-catalog/1.0.0 (iTop Hub)
InstalledExtension/combodo-sla-considering-business-hours/2.1.7 (Extension)
---- Actual modules installed ----
InstalledModule/approval-base: 3.0.1
InstalledModule/authent-cas: 2.7.0
InstalledModule/authent-external: 2.7.0
InstalledModule/authent-ldap: 2.7.0
InstalledModule/authent-local: 2.7.0
InstalledModule/combodo-autoclose-ticket: 1.0.1
InstalledModule/combodo-db-tools: 2.7.0
InstalledModule/combodo-email-synchro: 3.2.0
InstalledModule/itop-attachments: 2.7.0
InstalledModule/itop-backup: 2.7.0
InstalledModule/itop-config-mgmt: 2.7.0
InstalledModule/itop-config: 2.7.0
InstalledModule/itop-datacenter-mgmt: 2.7.0
InstalledModule/itop-endusers-devices: 2.7.0
InstalledModule/itop-files-information: 2.7.0
InstalledModule/itop-hub-connector: 2.7.0
InstalledModule/itop-ldap-collector: 1.0.1
InstalledModule/itop-object-copier: 1.3.2
InstalledModule/itop-portal-base: 2.7.0
InstalledModule/itop-portal: 2.7.0
InstalledModule/itop-profiles-itil: 2.7.0
InstalledModule/itop-sla-computation: 2.7.0
InstalledModule/itop-standard-email-synchro: 3.1.0
InstalledModule/itop-stencils: 1.0.6
InstalledModule/itop-storage-mgmt: 2.7.0
InstalledModule/itop-tickets: 2.7.0
InstalledModule/itop-virtualization-mgmt: 2.7.0
InstalledModule/itop-welcome-itil: 2.7.0
InstalledModule/molkobain-handy-framework: 1.3.0
InstalledModule/molkobain-newsroom-provider: 1.1.0
InstalledModule/molkobain-portal-mosaic-service-catalog: 1.0.0
InstalledModule/templates-base: 3.0.17
InstalledModule/contact_ticket: 1.0.0
InstalledModule/customer-survey: 2.4.2
InstalledModule/email-reply: 1.2.0
InstalledModule/itop-admin-delegation-profiles: 0.1.0
InstalledModule/itop-approval-portal: 1.0.4
InstalledModule/itop-bridge-virtualization-storage: 2.7.0
InstalledModule/itop-change-mgmt-itil: 2.7.0
InstalledModule/itop-core-update: 2.7.0
InstalledModule/itop-incident-mgmt-itil: 2.7.0
InstalledModule/itop-knownerror-mgmt: 2.7.0
InstalledModule/itop-problem-mgmt: 2.7.0
InstalledModule/itop-request-mgmt-itil: 2.7.0
InstalledModule/itop-service-mgmt: 2.7.0
InstalledModule/itop-synchro-dashboard: 1.0.0
InstalledModule/molkobain-console-tooltips: 1.1.2
InstalledModule/precanned-replies: 1.1.5
InstalledModule/samplefield_inciden-module: 1.0.0
InstalledModule/WorkOrderTemplate: 1.0.0
InstalledModule/combodo-dispatch-incident: 1.1.4
InstalledModule/combodo-dispatch-userrequest: 1.1.5
InstalledModule/combodo-sla-computation: 2.3.0
InstalledModule/fieldproject1-module: 1.0.0
InstalledModule/fieldservicefamily-module: 1.0.0
InstalledModule/itop-communications: 1.1.0
InstalledModule/itop-full-itil: 2.7.0
InstalledModule/itop-request-template: 2.0.5
InstalledModule/precanned-replies-pro: 1.0.5
InstalledModule/sv-auto-dispatch: 1.0.1
InstalledModule/combodo-approval-extended: 1.3.0
InstalledModule/combodo-coverage-windows-computation: 2.0.2
InstalledModule/itop-request-template-portal: 1.0.0
===== end =====

**Comment by pmendoza12** [comment]
After looking at the list configuration, Now i can see the Dispatch rules column (see picture attached) at the Customer Contract / Services TAB. 
But where is the dispatch rule supposed to be showed up?  with the service tab?, with the contract tab? , I have reviewed the services tabs, contracts, family of services, but no luck?.

**Comment by pmendoza12** [comment]
![image](https://user-images.githubusercontent.com/66842394/87342564-7837d280-c519-11ea-9288-8af515864fd2.png)

![customer contract](https://user-images.githubusercontent.com/66842394/87342679-a9180780-c519-11ea-922f-bdefa9745dba.JPG)

**Comment by Hipska** [comment]
Indeed, as I said on the customer contract details it should be in the services tab as you figured out. On the Service details it should be on the customer contracts tab.

So it indeed seems that another module is redefining the fields on that list, there is not much we can do about that.
I had a look at your extensions and it seems that combodo-coverage-windows-computation is to blame. Most recent version is 2.0.6 and this problem was fixed in 2.0.5 released april 26 '19. You are using version 2.0.2, contact Combodo for a newer version.

**Comment by pmendoza12** [comment]
Thanks a lot for your help and time, I really appreciate it !!!

Best Regards..

Ing. Pedro Mendoza
Jefe Unidad de Sistemas de Información
+58 212 9815849
+58 412 2264065
www.inelectra.com
